examples/android/binder/java/io/grpc/binder/cpp/example/native.cc

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021 gRPC authors.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <android/log.h>
+#include <jni.h>
+
+extern "C" JNIEXPORT jstring JNICALL
+Java_io_grpc_binder_cpp_example_ButtonPressHandler_native_1entry(
```

Comment:
Not familiar with Java JNI, but is this a typo? (`native_1entry` -> `native_entry`)

Reply:
Oh my bad, underscore in function name should be followed by a 1.